tests/test_embeds.py

```diff
@@ -0,0 +1,268 @@
+import io
+from datetime import datetime, timedelta
+
+import pytest
+
+from disnake import Color, Embed, File
+from disnake.utils import MISSING, utcnow
+
+_BASE = {"type": "rich"}
+
+
+@pytest.fixture
+def embed() -> Embed:
```

Comment:
It's not even a valid video embed, the main idea was to have an embed with all init fields set to non-default values. I'll change it to `"type": "link"` though since it would be a valid link embed.

Reply:
thoughts on naming it link_embed?